src/hooks.js

```diff
@@ -20,12 +20,12 @@ export const useFilecoin = () => {
     const getAccounts = async () => {
       try {
         const addresses = await filecoin.wallet.getAccounts();
-        const accounts = await Promise.all(addresses.map(address => {
-          return new Promise(async (resolve, reject) => {
-            const balance = await filecoin.getBalance(address)
-            resolve({ balance: balance, address: address })
-          });
-        }))
+        const accounts = await Promise.all(
+          addresses.map(async address => {
```

Comment:
Just removed one extra promise returned by this mapping (you're returning a new promise, and then performing another async action _inside_ the promise). This method isn't even anymore performant, just cleaner and more conventional.